core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/DataFeedService.java

```diff
@@ -39,14 +41,22 @@ public interface DataFeedService {
      * Parses the data feed to the list of resources({@link Resource}) for further adapting them to view models
      * and displaying in the Link Inspector grid. The number of output items is limited for the sake of UX consistency.
      *
+     * @param page - page number for displaying
      * @return the list of resources({@link Resource}) based on the data feed
      */
-    List<Resource> dataFeedToResources();
+    List<Resource> dataFeedToResources(int page);
 
     /**
      * Parses the data feed to the list of models({@link GridResource}). The number of output items is not limited.
      *
      * @return the list of view items({@link GridResource}) based on the data feed
      */
-    List<GridResource> dataFeedToGridResources();
+    List<List<GridResource>> dataFeedToGridResources();
```

Comment:
(To discuss) Can we do without a "list of lists"? The structure looks a bit ugly. Especially as a return value of a public service method.
If we cannot, then we need to modify Javadoc at least)